packages/cli/src/codemods/v0-8/ui-package-split.ts

```
@@ -23,49 +23,181 @@ const reactUIExports = [
+        arg &&
+        (arg.type === "Literal" || arg.type === "StringLiteral") &&
+        arg.value === "@assistant-ui/react/tailwindcss"
+      );
+    })
+    .forEach((path: any) => {
+      path.value.arguments[0].value = "@assistant-ui/react-ui/tailwindcss";
+      markAsChanged();
+      if (removedMarkdownPlugin) {
+        if (
+          path.parentPath &&
+          path.parentPath.value &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)